packages/manager/modules/pci/src/projects/project/private-networks/private-networks.controller.js

```diff
@@ -31,4 +41,59 @@ export default class {
   refreshMessages() {
     this.messages = this.messageHandler.getMessages();
   }
+
+  expandRow($row) {
+    if (
+      this.isRowExpanded[$row.id] === undefined ||
+      this.isRowExpanded[$row.id] !== true
```

Comment:
I think is same that

```suggestion
      this.isRowExpanded[$row.id] != true
```